app/presenters/organisations/what_we_do_presenter.rb

```diff
@@ -0,0 +1,43 @@
+module Organisations
+  class WhatWeDoPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def share_links
+      links = []
+
+      org.social_media_links.each do |link|
+        links << {
+          href: link["href"],
+          text: link["title"],
+          icon: link["service_type"]
+        }
+      end
+
+      {
+        stacked: true,
+        brand: org.brand,
+        links: links
+      }
+    end
+
+    def what_we_do_title
```

Comment:
Think this can just be called 'title', as when we reference it in  the view it's already prefixed with: `@what_we_do`

Reply:
Actually this is now duplicate code as it's been put into the much more generic show_presenter, so I'll remove it.